packages/fx-core/resource/package.nls.json

```diff
@@ -395,6 +395,7 @@
   "error.aad.manifest.OptionalClaimsIsMissing": "optionalClaims is missing\n",
   "error.aad.manifest.OptionalClaimsMissingIdtypClaim": "optionalClaims access token doesn't contain idtyp claim\n",
   "error.aad.manifest.AADManifestIssues": "AAD manifest has following issues that may potentially break the Teams App:\n",
+  "error.aad.manifest.AADPermissionIssues": "Update Azure Active Directory app failed. Please double check the permission id you specified is correct or try again later.\n",
```

Comment:
`try again later` should not help fix the issue. We can remove it.

Reply:
updated.